rtt_ros/src/orocos/types/ros_float32_type.cpp

```diff
@@ -1,38 +1,36 @@
-*                    Department of Mechanical Engineering,                    *
-*                   Katholieke Universiteit Leuven, Belgium.                  *
-*                                                                             *
-*       You may redistribute this software and/or modify it under either the  *
-*       terms of the GNU Lesser General Public License version 2.1 (LGPLv2.1  *
-*       <http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html>) or (at your *
-*       discretion) of the Modified BSD License:                              *
-*       Redistribution and use in source and binary forms, with or without    *
-*       modification, are permitted provided that the following conditions    *
-*       are met:                                                              *
-*       1. Redistributions of source code must retain the above copyright     *
-*       notice, this list of conditions and the following disclaimer.         *
-*       2. Redistributions in binary form must reproduce the above copyright  *
-*       notice, this list of conditions and the following disclaimer in the   *
-*       documentation and/or other materials provided with the distribution.  *
-*       3. The name of the author may not be used to endorse or promote       *
-*       products derived from this software without specific prior written    *
-*       permission.                                                           *
-*       THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR  *
-*       IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED        *
-*       WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE    *
-*       ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,*
-*       INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES    *
-*       (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS       *
-*       OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) *
-*       HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,   *
-*       STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING *
-*       IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE    *
-*       POSSIBILITY OF SUCH DAMAGE.                                           *
-*                                                                             *
-*******************************************************************************/
+ *                 This file is part of the OROCOS toolchain ROS project       *
+ *                                                                             *
+ *        (C) 2010 Steven Bellens, steven.bellens@mech.kuleuven.be             *
+ *                    Department of Mechanical Engineering,                    *
+ *                   Katholieke Universiteit Leuven, Belgium.                  *
```

Comment:
nit: Either we omit the trailing `*` for the whole copyright header, or add it as a box enclosing the whole license text with the correct width (without the additional space you introduced here)?

Suggestion:
```cpp
/*
 * This file is part of the OROCOS toolchain ROS project
 *
 * Copyright (C) 2010 Steven Bellens, steven.bellens@mech.kuleuven.be
 * Department of Mechanical Engineering,
 * Katholieke Universiteit Leuven, Belgium.
 *
 * ...
 */
```

following the example from [roscpp](https://github.com/ros/ros_comm/blob/kinetic-devel/clients/roscpp/include/ros/advertise_options.h), or

```cpp
// This file is part of the OROCOS toolchain ROS project
//
// Copyright (C) 2010 Steven Bellens, steven.bellens@mech.kuleuven.be
// Department of Mechanical Engineering,
// Katholieke Universiteit Leuven, Belgium.
//
// ...
```
which is the format applied in [rclcpp](https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/node.hpp) and many other ROS 2 packages, for example.

Either way is fine as long as it is somehow consistent across the whole repo.